theorems/T000659.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: T000659
+if:
+  and:
+  - P000208: true
+  - P000134: true
+then:
+  P000185: true
+---
+
+Consider its Kolmogorov quotient $\mathrm{KQ}(X)$, it is {P3}, thus {P100}. Now it follows by {P208} that every subspace is closed, which implies {P52}.
+
+Therefore, $\mathrm{KQ}(X)$ is {P52}, and we obtain that $X$ is {P185}.
```

Comment:
```suggestion
First observe that a {P208} {P3} space is {P52}, since every subset is compact, hence closed.

Now, if $X$ is {P208} and {P134}, its Kolmogorov quotient is {P208} and {P3}, hence {P52}.  This is equivalent to $X$ being {P185}.
```
Rewrite to make things clearer.